### ISSUE TITLE
Arbitrary resolution

### ISSUE THREAD

**Issue by HyoKong** [Doubiiu/DynamiCrafter#22]
Hi, thank you so much for your great work!

May I ask how to adapt to images with arbitrary resolutions, for instance, 512x768?

Thank you so much for your help!

**Comment by HyoKong** [comment]
Hi, adjusting height may be not working. there are bugs when concatenating the output of downsampling and upsampling blocks.  The shapes do not match. 

https://github.com/Doubiiu/DynamiCrafter/blob/ce86a1207494e11f7befda7bfb5235ea8f9d518e/lvdm/modules/networks/openaimodel3d.py#L596

Could you pls help to solve this issue?

**Comment by Doubiiu** [comment]
Hi, what is your exact resolution for inference? To my best knowledge, the height and width should be divided by 32 (I think this is the reason of your bug). And can you summarize the change of code for the target resolution inference?
